.readthedocs.yml

```diff
@@ -1,11 +1,11 @@
 
 version: 2
 
-formats: all
+formats: [htmlzip]
 
 python:
   system_packages: true
```

Comment:
should be removed - this option is deprecated (similar to https://github.com/radical-cybertools/radical.pilot/pull/3029)